scripts/makedistro.bat

```diff
@@ -6,4 +6,4 @@ pip install --upgrade -r requirements.txt
 pip install --upgrade openai
 pip install --upgrade google.generativeai
 pip install --upgrade anthropic
-pyinstaller --noconfirm --additional-hooks-dir="PySubtitleHooks" --add-data "theme/*;theme/" --add-data "instructions*;." --add-data "LICENSE;." --add-data "gui-subtrans.ico;." gui-subtrans.py
+pyinstaller --noconfirm --additional-hooks-dir="PySubtitleHooks" --add-data "theme/*;theme/" --add-data "instructions*;." --add-data "LICENSE;." --add-data "assets/gui-subtrans.ico;." gui-subtrans.py
```

Comment:
I think the add-data for instructions will need modifying to package them in a subdirectory... presumably something similar to theme (subtly different for windows/mac for no clear reason)

Reply:
What do you think, should we move it to subdirectory?

Reply:
Yes, I think it's a good idea to put them in a subdirectory :)

Reply:
Could you check this out that does the modifying need something to work? I mean, if you like in general would like to see this pr-merged at some point

Reply:
I'll merge it now and fix up anything that needs fixing when I package the next release :-)

Reply:
Do you want to move the scripts back to root first, so we can tackle them separately later?